### ISSUE TITLE
Set up new branch for python 3 port

### ISSUE THREAD

**Issue by squiddy** [unknown-horizons/unknown-horizons#2631]
But keep the old one around for reference.

**Comment by AndyMender** [comment]
I opened a new branch called "py2_to_py3" and pushed it to my fork. Unless you would like to open a branch yourself, I think that's alright, yes?

The most important question, though is **which version of Python3?**.
The most recent version is Python3.6, but Ubuntu 16.04 LTS and Debian Stable have Python3.4 n their repositories. Luckily, FreeBSD offers Python3.4, 3.5 and 3.6 so I'm not really limited :).

**Comment by squiddy** [comment]
Well, I did say I was going to open a branch (since you asked me), but since you've started and I'll have to run some errands, I'd say go ahead. ;)

At some point we should move the branch into this repository though, then we both can work on it, submitting PRs etc.

For now we should stick to 3.4. I doubt we need anything from 3.5-3.6 to make this port work.

**Comment by AndyMender** [comment]
I will then stick with the branch I created and when I have something worthwhile, we can move it / merge it into the UH repository. 

There are some improvements in 3.5 that might be useful (additional list/tuple unpacking contexts), however they're so minor that we should not concern ourselves with them most likely. Not sure about 3.6, since I only skimmed through the release notes at one point and didn't find anything ground-breaking there.
